test/blackbox-tests/test-cases/utop/utop-executables.t/bin/dune

```diff
@@ -0,0 +1,3 @@
+(executable
+ (name foo)
+ (libraries ppxlib))
```

Comment:
To make the test more independent, could you switch to a locally defined dummy library instead of ppxlib?

Reply:
Sure, done.